### PR TITLE
Restyle/brand old databrowser

### PR DESCRIPTION
**Pull request by ericyoungberg**
This PR will apply a new skin to databrowser that is inline with the branding for the new DataCommon.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,5 @@ testem.log
 .node_modules.ember-try/
 bower.json.ember-try
 package.json.ember-try
+
+*.sublime-project
```
Comment (mzagaja) [inline .gitignore:25]: Sublime project files are supposed to be checked into version control per https://www.sublimetext.com/docs/3/projects.html

**Review by mzagaja — DISMISSED** [review]
This seems mostly good to me. However, the test did not pass. I came up with a test that demonstrates the component displays appropriate placeholder text:

```
import { module, test } from 'qunit';
import { setupRenderingTest } from 'ember-qunit';
import { render } from '@ember/test-helpers';
import hbs from 'htmlbars-inline-precompile';
import EmberObject from '@ember/object';
import ArrayProxy from '@ember/array/proxy';
import { A } from '@ember/array';

module('Integration | Component | search-bar', function(hooks) {
  setupRenderingTest(hooks);

  let datasets = [
    { id: '1', title: 'Households with Kids' },
    { id: '2', title: 'Households without Kids' }
    ];

  test('it renders', async function(assert) {
    // Set any properties with this.set('myProperty', 'value');
    // Handle any actions with this.set('myAction', function(val) { ... });

    this.set('searchables', datasets)

    await render(hbs`
      {{search-bar records=searchables class="lift"}}
    `);

    assert.equal(this.element.children[0].children[0].placeholder, 'Search 2 datasets ...');
  });
});
```
Tomorrow morning maybe we can do one that tests the input into the component and making sure it returns what we want. Let me know if this makes sense.

File: .gitignore (hunk 1 above)
Comment (ericyoungberg) [reply .gitignore:25]: But sublime project files have nothing to do with the project and only for sublime's sake

Comment (mzagaja) [reply .gitignore:25]: On a functional level it makes my life easier by not having to rebuild the project file for sublime on my home computer after making it on my work computer and vice-versa as it is some settings relevant only to the specific project but for sublime. If you feel strongly about this I will accept this change and go through the extra hassle, but if it does not cause you too much pain it'd be great if we could leave this in.

**Pull request merged**